WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsView.swift

```diff
@@ -151,6 +79,119 @@ struct WooShippingCreateLabelsView: View {
                 }
             }
         }
+        .onAppear() {
+            guard viewModel.storeOptions == nil else { return }
+            viewModel.loadStoreOptions()
+        }
+    }
+
+    @ViewBuilder
+    private var expandableBottomSheet: some View {
```

Comment:
Separated UI code into helper properties so it can be updated easier in the future